assayist/processor/utils.py

```diff
@@ -186,6 +186,9 @@ def download_source(build_info, output_dir):
     log.info(f'Downloading source for {build_info["id"]}')
 
     url, _, commit_id = source_url.partition('#')
+    # Sometimes the source URL uses "?#" to separate the repo from the commit, so we can just strip
+    # the unneeded question mark
+    url = url.rstrip('?')
```

Comment:
LGTM, 

Just a general question. Do you know what is a use-case for that question mark in git URI?

Reply:
Not sure, but it seems unnecessary in this case.